mdbenchmark/submit.py

```diff
@@ -75,39 +85,51 @@ def submit(directory, force_restart):
     if not bundle:
         console.error("No benchmarks found.")
 
-    grouped_bundles = bundle.categories.groupby("started")
-    try:
-        bundles_not_yet_started = grouped_bundles[False]
-    except KeyError:
-        bundles_not_yet_started = None
-
-    if not bundles_not_yet_started and not force_restart:
-        console.error(
-            "All generated benchmarks were already started once. "
-            "You can force a restart with {}.",
-            "--force",
+    df = MakeDataFrameFromBundle(bundle)
+    # Reformat NaN values nicely into question marks.
+    df_to_print = df.replace(np.nan, "?")
+    df_to_print = df.drop(columns=['ns/day','ncores'])
+    with pd.option_context("display.max_rows", None):
+        print(df_to_print)
+
+    # here I add the user promt to confirm the submission of the simulations
+    if yes or click.confirm('The above simulations will be submitted. Continue?'):
```

Comment:
Invert the conditions here. This way we exit early and the biggest part of the code (the currently following lines) will not be indented.

```python
if not yes or not click.confirm('The above simulations will be submitted. Continue?'):
    console.warn("Exiting. No benchmarks submitted.")

grouped_bundles = bundle.categories.groupby("started")
[...]
```